monkey/infection_monkey/Pipfile

```diff
@@ -17,11 +14,8 @@ netifaces = ">=0.10.9"
 odict = "==1.7.0"
 paramiko = ">=2.7.1"
 psutil = ">=5.7.0"
-pycryptodome = "==3.9.8"
-pyftpdlib = "==1.5.6"
 pymssql = "==2.1.5"
 pypykatz = "==0.3.12"
-pysmb = "==1.2.5"
```

Comment:
This is used by zerologon. See `infection_monkey/exploit/zerologon_utils/vuln_assessment.py`.

https://github.com/guardicore/monkey/blob/f457635507f34e8382a54f0f21ecdeca825c7aee/monkey/infection_monkey/exploit/zerologon_utils/vuln_assessment.py#L4